packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -159,7 +160,7 @@ export interface ContainerRuntimeMessage {
 }
 
 export interface IGeneratedSummaryData {
-    readonly summaryStats: ISummaryStats;
+    readonly summaryStats: ITopLevelSummaryStats | ISummaryStats;
```

Comment:
Why is this a union type? When would we ever return `ISummaryStats` now?

Reply:
it's in the summarizer.spec.ts , let me change that